td4m3/src/main/java/ca/td/td4m3/components/mortgage/binder/MortgageBinder.kt

```diff
@@ -18,5 +18,12 @@ class MortgageBinder(@DrawableRes private val drawableId: Int,
         viewHolder.imageView.setBackgroundResource(drawableId)
         viewHolder.imageView.invalidate()
         viewHolder.imageView.setOnClickListener(listener)
+
+        if (title.isEmpty()) {
+            viewHolder.textView.visibility = View.GONE
+        } else {
+            viewHolder.textView.text = title
```

Comment:
viewHolder.textView.visibility = View.Visible (or whatever it's called)

Reply:
Done.